build.gradle.kts

```diff
@@ -131,7 +131,7 @@ tasks.register("buildNdkIntegrationTestsArtifacts") {
     dependsOn(":instrumented:integration:assembleDebug")
 }
 
-nightlyTestsCoverageConfig(threshold = 0.85f)
+nightlyTestsCoverageConfig(threshold = 0.81f)
```

Comment:
A task was added to cover these:
https://datadoghq.atlassian.net/browse/RUM-3389